client/apps/user-portal/src/app/sermons/sermon-view/sermon-view.component.ts

```diff
@@ -1,13 +1,18 @@
 import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { Observable } from 'rxjs';
 
 @Component({
   selector: 'app-sermon-view',
   templateUrl: './sermon-view.component.html',
   styleUrls: ['./sermon-view.component.scss']
 })
 export class SermonViewComponent implements OnInit {
+  // sermon$: Observable<Sermon>;
```

Comment:
Why did you comment out the `sermon$` Observable?

We should have this, somewhat ready to use once we have data.
Hardcoding things in the template will not help with that.

Can you update to include the mocked out Observable, please?

Reply:
Do we need to setup interpolation for the template, like on the sermon title, date, etc.?  That is part of the concern right?

Reply:
So, does the interpolation in the html file need to setup or no?